### PR TITLE
Update wording on the incentive page

### PR DESCRIPTION
**Pull request by LucasCairns**

### CODE REVIEW

File: server/views/components/personal-incentives/template.njk [hunk 1]
```diff
@@ -23,20 +23,20 @@
...
-          Your incentives level is unavailable
+          Your incentive level is unavailable
         </p>
       {% endif %}
       <p class="incentivesReview">
         {% if params.data.reviewDate %}
         Your level can be reviewed on <span>{{ params.data.reviewDate }}</span>
         {% else %}
-        Your incentives review date is unavailable
+        Your incentive review date is unavailable
```
Comment (psoleckimoj) [inline server/views/components/personal-incentives/template.njk:33]: I'd say `incentive level review data`?

Comment (psoleckimoj) [reply server/views/components/personal-incentives/template.njk:33]: *date

File: server/services/offender/index.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ const createOffenderService = (
       );
       logger.debug(e.stack);
       return {
-        error: 'We are not able to show your incentives summary at this time',
+        error: 'We are not able to show your incentive summary at this time',
```
Comment (psoleckimoj) [inline server/services/offender/index.js:36]: Also this I'd add level just to be absolutely clear.

Comment (psoleckimoj) [reply server/services/offender/index.js:36]: @LucasCairns just this one I think?

Comment (LucasCairns) [reply server/services/offender/index.js]: updated 👍

**Review by psoleckimoj — APPROVED** [review]
LGTM

**Pull request merged**